Mathlib/Tactic/Linter/Multigoal.lean

```diff
@@ -0,0 +1,161 @@
+/-
+Copyright (c) 2024 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import Lean.Elab.Command
+
+/-!
+#  The "multiGoal" linter
+
+The "multiGoal" linter emits a warning where there is more than a single goal in scope.
+There is an exception: a tactic that closes *all* remaining goals is allowed.
+
+There are a few tactics that are intended to work specifically in such a situation and the linter
```

Comment:
This is getting into copy-editing territory: this description covers everything - but re-reading this again (after a few months), I find this hard to parse. There are exceptions, and exceptions of exceptions... can the same content be said more clearly and directly? Something like:

```
Suppose there are multiple goals in scope, and the next tactic call does not close them all. (This is a common situation, as tactic can split one goal into several steps, or to leave side conditions to be checked.)

There are a few tactics (such as `skip`, `skip` or the `try` combinator) that are intended to work specifically in such a situation. Otherwise, the mathlib style guide ask that goals be be focused until there is only one active goal at a time.
If this focusing does not happen, the linter emits a warning.
Typically, the focusing is achieved by the `cdot`: `·`, but, e.g., `focus` or `on_goal x` also serve a similar purpose.
```

This is longer and less concise, but (I think) clearer: less complicated sentences, less negation